src/pymovements/gaze/io.py

```diff
@@ -360,6 +362,98 @@ def from_asc(
             for fileinfo_key, fileinfo_dtype in column_schema_overrides.items()
         ])
 
+    if experiment is None:
+        experiment = Experiment(sampling_rate=metadata['sampling_rate'])
+    if experiment.screen is None:
+        experiment.screen = Screen()
+    if experiment.eyetracker is None:
```

Comment:
Good catch! You actually exposed a type ambivalence in the `Experiment.eyetracker` field here. In contrast to the `screen` field, the `eyetracker` field can be `None`. In this case you should keep the if-clause in this PR, as the current implementation in `Experiment.eyetracker` is indeed type-ambivalent.

In a follow-up the `eyetracker` field could be improved in a similar way as I suggested with the `screen` field. Feel free to create a short initial issue for this where we can develop that idea further.

Reply:
#891